README.md

```diff
@@ -114,6 +114,9 @@ An example cURL command to start the connector in distributed mode is:
     data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64", "channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
     ```
 
+## Breaking API Changes in Version 2.0.0
+
+Please see our [Upgrade / Migration Guide](UPDATING.md) for notes on changes you need to make to your configuration to update it with changes introduced by version 2.0.0 of the connector.
```

Comment:
Please always include vertical whitespace around headings... so you need an empty line below this para before the 'Configuration' H2.

Reply:
Good point :) bfb40948a5b9d670644ae23fc5c2a9910b123289